src/textual/dom.py

```diff
@@ -888,6 +905,15 @@ def set_class(self, add: bool, *class_names: str) -> None:
         else:
             self.remove_class(*class_names)
 
+    def set_classes(self, classes: str | Iterable[str]) -> None:
```

Comment:
Perhaps this should return `self` so folk can chain calls?

Reply:
It probably should. We don't have that on the other classes either.